technical-reference/reference-technical-articles/multichain-governance.md

```diff
@@ -1,7 +1,7 @@
 # Multichain Governance
 
 {% hint style="warning" %}
-**To be released**
+**Only available on BNB chain, Ethereum and opBNB**
 {% endhint %}
```

Comment:
I would remove this, because it will be available on every supported network

Reply:
https://github.com/VenusProtocol/venus-protocol-documentation/pull/208/commits/3b5df8b39b7652d63b6ace06d235702670812320